test-data/cluster-hoprd.yaml

```diff
@@ -9,13 +9,13 @@ metadata:
 spec:
   identityPoolName: pull-requests-rotsee
   replicas: 1
-  version: saint-louis-latest
+  version: singapore-latest
   supportedRelease: saint-louis
   enabled: true
   service:
     type: LoadBalancer
   config: |
     hopr:
       chain:
-        network: doufur
-        provider: https://rpc.ankr.com/gnosis
+        network: rotsee
+        provider: https://gnosis.rpc-provider.prod.hoprnet.link
```

Comment:
I would not put our internal resources as defaults for a public repo.

Reply:
Changed